scripts/add-new-dex.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
+set -euxo pipefail
+
+# Add new DEX.
+# Extending the existing Hermes settings and creating a connection between Nolus and the new DEX.
+
+SCRIPTS_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+source "$SCRIPTS_DIR"/internal/verify.sh
+source "$SCRIPTS_DIR"/internal/add-dex-support.sh
+
+NOLUS_NET="http://localhost:26612/"
+NOLUS_HOME_DIR="$HOME/.nolus"
+WALLET_WITH_FUNDS_KEY="reserve"
+NOLUS_CHAIN_ID=$(grep -oP 'chain-id = "\K[^"]+' "$NOLUS_HOME_DIR"/config/client.toml)
+
+HERMES_CONFIG_DIR_PATH="$HOME/.hermes"
+HERMES_BINARY_DIR_PATH="$HOME/hermes"
+CHAIN_ID=""
+CHAIN_IP_ADDR_RPC=""
+CHAIN_IP_ADDR_GRPC=""
+CHAIN_ACCOUNT_PREFIX=""
+CHAIN_PRICE_DENOM=""
+CHAIN_TRUSTING_PERIOD=""
+
+
+while [[ $# -gt 0 ]]; do
+  key="$1"
+
+  case $key in
+
+  -h | --help)
+    printf \
+    "Usage: %s
+    [--nolus-home-dir <nolus_home_dir>]
+    [--wallet-with-funds-key <nolus_wallet_with_funds_key>]
```

Comment:
it'd be better to rename it to something more specific and related to its role in the process